docs/release/trg-1/trg-1-08.md

```diff
@@ -0,0 +1,26 @@
+---
+title: TRG 1.08 - Interface documentation (APIs)
+sidebar_position: 1
+---
+
+| Status | Created     | Post-History                                          |
+|--------|-------------|-------------------------------------------------------|
+| Active | 22-Apr-2024 |                                                       |
+| Draft  | 21-Mar-2024 | Transferred from Release Acceptance Criteria into TRG |
+
+## Why
+
+Interface documentation is crucial to interact with components. It fundamentally enables others to consume your APIs.
+
+## Description
+
+- The interface documentation **must** be stored in the folder `/docs/api` as `openAPI.yaml`
+  - It **must** be in the OpenAPI standard
+  - It **must** contain all interfaces
+  - It **must** be up-to-date for the latest released component version
+
+## Best Practice
+
+- consider functional and non-functional commitments of each interface
```

Comment:
Could you explain, what is meant by that?

Reply:
Its basically a reminder in sense of "think about what this interface actually does regarding functional and non functional requirements" Basically a 'rethink it.

@RolaH1t you wanted to ask about this sentence right?

Reply:
@SebastianBezold can we keep it as it is for now?

Reply:
Fine with me. Would still like to know, if committers, that are going to stay do know what to check, or what to recommend for this beset-practice